spark-jobs/src/main/scala/za/co/absa/enceladus/common/plugin/menas/EventListenerMenas.scala

```diff
@@ -132,4 +151,24 @@ class EventListenerMenas(dao: MenasDAO,
       true
     }
   }
+
+  private def notifyPlugins(): Unit = {
+    val additionalParams = Map[String, String]("datasetName" -> datasetName,
+      "datasetVersion" -> datasetVersion.toString,
+      "reportDate" -> reportDate,
+      "reportVersion" -> reportVersion.toString,
+      "runStatus" -> _runStatus.toString
+    )
+    _controlMeasure.foreach(measures =>
```

Comment:
Tiny: Considering the `var` is called `_controlMeasure` I would call the used variable just `measure`.

Reply:
Since `_controlMeasure` has type of `ControlMeasure` the less entropy is introduced by actually having a longer name => feels more readable this way.

Reply:
Really it feels more readable to you, that a `controlMeasure`'s collection item is measures? :-)
But not worth to dwell over it and definitely doe not justify an extra commit because of it. ;-)

Reply:
Oh, sorry. I see your point. Since `_controlMeasure` is an Option, the inner variable is better to be called `measure`, I agree. It deserves another commit :)